travis/push-image.sh

```diff
@@ -21,6 +21,9 @@ if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_PULL_REQUEST" = "false" ]]; then
     # tag temporarily as liberoadmin due to lack of `libero/` availability
```

Comment:
We should contact them again.

Reply:
We have a minimal paid plan on `elifesciences`, maybe that will help?

Reply:
💵